### DISCUSSION TITLE
Android设备无法使用ws+tls的配置

### DISCUSSION THREAD

**Discussion by wooodypan** [FranzKafkaYu/x-ui#380] [Q&A] [answered]
Android设备使用的App是v2rayNG，已测试了vmess+ws+tls和vless+ws+tls，均无法使用，关闭tls选项可以，想问下可能是什么原因导致的？同一配置macOS和iOS是可以用的

**Comment by FranzKafkaYu** [comment]
V2rayN可以正常使用么，通常是证书配置问题，可以尝试在v2rayNG中跳过证书验证试试，如跳过之后可以使用，则请在服务端证书配置中使用全链证书

**Reply by wooodypan** [reply]
感谢回复，v2rayNG设置了跳过证书验证还是不行，但是另外一个应用Surfboard可以（Surfboard目前不支持vless），我用vmess+ws+tls试的

**Reply by FranzKafkaYu** [reply, marked as answer]
> 感谢回复，v2rayNG设置了跳过证书验证还是不行，但是另外一个应用Surfboard可以（Surfboard目前不支持vless），我用vmess+ws+tls试的

确认一下服务端版本与客户端版本信息，确保xray core版本一致.总的来讲使用VMESS ws tls需要注意三点：alterId需要保持为0；服务端与客户端时间需要保持同步（时区没有影响）;尽量使用全链证书。
